### PR TITLE
update release SOP document

### PR DESCRIPTION
**Pull request by sabrinatoro**
- updated the command for preparing the release
- updated (hopefully) some formating

### CODE REVIEW

**Review by matentzn — APPROVED** [review]
LGTM! (Looks good to me)

File: docs/developer-guide/release.md [hunk 1]
```diff
@@ -105,19 +102,19 @@ Press Enter to open github.com in your browser...
...
-   1. Add the release description to the release tab:
-       - All of the releases can be found under the [releases](https://github.com/monarch-initiative/mondo/releases) tab.
-       - To add a description of the release:
-         - click edit
-         - in the 'describe this release' section add the content from the file `src/ontology/reports/mondo_release_diff.md`.
+    1. Add the release description to the release tab:
+          - All of the releases can be found under the [releases](https://github.com/monarch-initiative/mondo/releases) tab.
+          - To add a description of the release:
+                - click edit
+                - in the 'describe this release' section add the content from the file `src/ontology/reports/mondo_release_diff.md`.
```
Comment (twhetzel) [inline docs/developer-guide/release.md:109]: I don't see this file `src/ontology/reports/mondo_release_diff.md`.

**Review by twhetzel — COMMENTED** [review]
Two comments inline.

File: docs/developer-guide/release.md [hunk 2]
```diff
@@ -47,17 +47,14 @@ _Note: While the release is running, don't shut your laptop or switch between re
 4. Merge the PR and continue with the release.
 
 ## Prepare the release
-1. Do a docker pull: `docker pull obolibrary/odkfull`
+1. Do a docker pull: `docker pull obolibrary/odkfull:dev`
```
[the remark this answers is not in the document]
Comment (matentzn) [reply docs/developer-guide/release.md:50]: I manage the dev image.. due to our numerous Python dependencies we need to constantly update our image. The dev image always has all the up to date Python dependencies.

File: docs/developer-guide/release.md (hunk 1 above)
Comment (matentzn) [reply docs/developer-guide/release.md:109]: It is in gitignore you will only see it if you are the one running the release!

File: docs/developer-guide/release.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (twhetzel) [reply docs/developer-guide/release.md:50]: a brief note about that would be useful to add

File: docs/developer-guide/release.md (hunk 1 above)
Comment (twhetzel) [reply docs/developer-guide/release.md:109]: Gotcha! That would be good to note in the docs.

Comment (sabrinatoro) [reply docs/developer-guide/release.md]: updated

File: docs/developer-guide/release.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sabrinatoro) [reply docs/developer-guide/release.md:50]: updated

**Comment by sabrinatoro** [comment]
@twhetzel notes have been added as requested. Ready for your review.

**Review by twhetzel — APPROVED** [review]
lgtm

**Comment by twhetzel** [comment]
I can merge after QC checks pass.

**Pull request merged**